README.md

```diff
@@ -47,23 +45,13 @@ npm run android
 yarn android
```

Comment:
`yarn android` seems to be useless for me.
You can just launch `yarn start` - then strike `a` to launch the app on an android emulator.
But maybe I missed something.

Reply:
I'll retry on my side, I am almost sure I got issues without `yarn android`.